src/sentry/models/group.py

```diff
@@ -390,21 +374,7 @@ def get_score(self):
         return type(self).calculate_score(self.times_seen, self.last_seen)
 
     def get_latest_event(self):
```

Comment:
This may be a little tricker to remove.
I saw it is used in the plugin code:

> src/sentry/plugins/base/v1.py:        event = group.get_latest_event() or Event()
> src/sentry/plugins/bases/issue.py:        event = group.get_latest_event()
> src/sentry/plugins/bases/issue2.py:        event = group.get_latest_event()
> src/sentry/plugins/bases/issue2.py:        event = group.get_latest_event()
> src/sentry/tasks/email.py:    event = group.get_latest_event()

I can think about two possible issues:
1) load on snuba
2) the event may have not been stored in snuba while it would already been in postgres. postgres write happens before we execute plugins in post_process.py while at that point the event has just been produced in kafka. so it may or may not have been consumed yet. This should be verified.

Reply:
Re 2: We might get away with this since these all seem to be view related functions - the event object is passed to any post process plugin code so we don't need to do `.get_latest_event()` there

Reply:
Hmm, this might come up in slack unfurls

Reply:
Actually post_process is synchronized on clickhouse storage. So it should be fine https://github.com/getsentry/sentry/blob/master/src/sentry/eventstream/kafka/consumer.py#L119-L145